fuel-client/src/client/schema/snapshots/fuel_gql_client__client__schema__node_info__tests__node_settings_query_gql_output.snap

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
this snapshot doesn't have a corresponding test

Reply:
good catch, will remove it